docker/docker-compose.yml

```diff
@@ -11,11 +11,81 @@ volumes:
   mainflux-auth-db-volume:
   mainflux-users-db-volume:
   mainflux-things-db-volume:
+  mainflux-keto-db-volume:
   mainflux-auth-redis-volume:
   mainflux-es-redis-volume:
   mainflux-mqtt-broker-volume:
 
 services:
+  keto:
+    image: oryd/keto:v0.6.0-alpha.3
+    container_name: mainflux-keto
+    ports:
+      - ${MF_KETO_READ_REMOTE_PORT}:4466
+      - ${MF_KETO_WRITE_REMOTE_PORT}:4467
+    environment:
+      - DSN=postgresql://${MF_KETO_DB_USER}:${MF_KETO_DB_PASS}@keto-db:${MF_KETO_DB_PORT}/${MF_KETO_DB}?sslmode=disable
+    command: serve -c /home/ory/keto.yml
+    restart: on-failure
+    volumes:
+      - type: bind
+        source: ./keto
+        target: /home/ory
+    networks:
+      - mainflux-base-net
+    depends_on:
+      - keto-db
+      - keto-migrate
+
+  keto-init:
+    image: oryd/keto:v0.6.0-alpha.3
+    container_name: mainflux-keto-init
+    environment:
+      - KETO_WRITE_REMOTE=keto:${MF_KETO_WRITE_REMOTE_PORT}
+      - DSN=postgresql://${MF_KETO_DB_USER}:${MF_KETO_DB_PASS}@keto-db:${MF_KETO_DB_PORT}/${MF_KETO_DB}?sslmode=disable
+    volumes:
+      - type: bind
+        source: ./keto
+        target: /home/ory
+    command: relation-tuple create /home/ory/relation-tuples
+    restart: on-failure
+    networks:
+      - mainflux-base-net
+    depends_on:
+      - keto-db
+      - keto-migrate
+
+  keto-migrate:
```

Comment:
do we need to have `keto-init` and `keto-migrate` to be started every time composition is up

Reply:
Actually, successful `keto-init` and `keto-migrate` are sufficient for once. We do not need them to build every composition. They prepare the database for Keto.

Reply:
I would move `keto-init` and `keto-migrate` to separae docker-compose file and add explanation how to prepare DB for keto into README.md

Reply:
Agreed. This should be solved in the better way. Do we need these containers at all - are those maybe just binary tools that we can download in the script and execute? Anyway - they are burdening the main docker-compose and we need to find an elegant way to introduce Keto.